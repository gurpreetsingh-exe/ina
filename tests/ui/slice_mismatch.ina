// COMPILE_FAIL

fn main() {
    let _: i32 = [1, 2, 3];
    let x = [1, 2, 3];
    let y: &i32 = x;
}
