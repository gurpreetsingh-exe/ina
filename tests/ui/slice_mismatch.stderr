error: mismatch types
   ╭─[tests/ui/slice_mismatch.ina:4:18]
   │
 4 │     let _: i32 = [1, 2, 3];
   │                  ────┬────
   │                      ╰───── expected `i32`, found `[integer]`
   ∙
───╯
error: mismatch types
   ╭─[tests/ui/slice_mismatch.ina:6:19]
   │
 6 │     let y: &i32 = x;
   │                   ┬
   │                   ╰─ expected `&i32`, found `[integer]`
   ∙
───╯
